course_content/notebooks/numpy_intro.ipynb

```diff
@@ -1515,9 +1499,7 @@
     "\n",
     "### Concepts:\n",
     "\n",
-    "  * [masked arrays](https://docs.scipy.org/doc/numpy/reference/maskedarray.generic.html) and missing data:\n",
```

Comment:
Why were these removed?

Reply:
This too is [now covered in the main course content](http://nbviewer.jupyter.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1#Masked-Arrays).